server/common/utils.py

```diff
@@ -11,6 +11,7 @@
 from urllib.parse import urlsplit, urljoin
 import numpy as np
 import pandas as pd
+
```

Comment:
This file seems to only have blank lines added? Is this a linting thing?

Reply:
no I ran into some import errors so I think that is leftover from moving the get_secrets around. Will remove